README.md

```diff
@@ -22,12 +22,20 @@ Let's take Unity Catalog for spin. In this guide, we are going to do the followi
   as well as provide a convenient way to explore the content of any UC server implementation.
 
 ### Prerequisites
+- Docker
+
+OR
+
 You have to ensure that your local environment has the following:
 - Clone this repository.
 - Ensure the `JAVA_HOME` environment variable your terminal is configured to point to JDK11+.
 - Compile the project using `build/sbt package`
 
 ### Run the UC Server
+```shell
+docker run -d -i --name unitycatalog -p 8081:8081 unitycatalog/unitycatalog:0.1.0
```

Comment:
this docker run step will only work with published versions of the server? like 0.1? 
my goal was for these steps to be runnable by anyone with the current unpublished code as soon as they clone this repo

Reply:
We could document how to build the container locally, and afterwards also execute it locally.